picard/ui/options/tags.py

```diff
@@ -72,6 +88,10 @@ def load(self):
         self.ui.preserved_tags.update(config.setting['preserved_tags'])
         self.ui.preserved_tags.set_user_sortable(False)
 
+        # Load disable date sanitization formats
+        disabled = config.setting['disable_date_sanitization_formats'] or []
```

Comment:
```suggestion
        disabled = config.setting['disable_date_sanitization_formats']
```

`or []` shouldn't be needed (`ListOption`)